examples/directml/phi_1/microsoft_phi.py

```diff
@@ -0,0 +1,99 @@
+# from olive.workflows import run as olive_run
+# olive_run("phi_1_config.json")
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import argparse
+import json
+import os
+import shutil
+import sys
+from pathlib import Path
+
+import onnxruntime as ort
+from packaging import version
+from olive.workflows import run as olive_run
+
+# pylint: disable=redefined-outer-name
```

Comment:
Why is this needed?

Reply:
To disable the following warning:
Warning (PYLINT) W0621
    Redefining name 'script_dir' from outer scope (line 70) (redefined-outer-
    name)

Reply:
Sure, but the solution would be to rename the variable to not overwrite it then. I don't see where the redefining is happening either.